pages/comparison.vue

```diff
@@ -205,6 +217,17 @@ onUnmounted(() => {
 <style lang="scss" scoped>
 @use "sass:map";
 
+.results-cards-container {
+  // The widest width that might trigger the heading flex row to wrap
+  // (Determined empirically using longest heading 'Explore by global vaccine investment' and longest parameter values)
+  $comparison-heading-wrap-width: 91rem;
+
+  @media (min-width: $comparison-heading-wrap-width) {
+    position: relative;
+    top: -2.5rem;
+  }
```

Comment:
The purpose of this is to bump up the tabs when there is space available to do so, saving vertical space.